gymnasium/experimental/wrappers/__init__.py

```diff
@@ -99,21 +101,65 @@
 }
 
 
-def __getattr__(name: str):
-    """To avoid having to load all wrappers on `import gymnasium` with all of their extra modules.
+def __getattr__(wrapper_name: str):
+    """Load a wrapper by name.
 
-    This optimises the loading of gymnasium.
+    This optimises the loading of gymnasium wrappers by only loading the wrapper if it is used.
+    Errors will be raised if the wrapper does not exist or if the version is not the latest.
 
     Args:
-        name: The name of a wrapper to load
+        wrapper_name: The name of a wrapper to load.
 
     Returns:
-        Wrapper
+        The specified wrapper.
+
+    Raises:
+        AttributeError: If the wrapper does not exist.
+        InvalidVersionWrapper: If the version is not a digit or is greater than the latest version.
+        DeprecatedWrapper: If the version is not the latest.
     """
-    if name in _wrapper_to_class:
```

Comment:
Why are we removing this? Personally I think that we should first check if the full wrapper name exists in the dictionary of wrappers then do the future checks to see if the wrapper version is wrong etc

Reply:
The proposed solution handles the case where an user can mistype the version number like `ClipRewardV!`
```python
    if version < 0:
        raise InvalidVersionWrapper(
            f"{wrapper_name} is not a valid version number, use {latest_wrapper_name} instead."
        )
```

Reply:
But I'm fine removing it if it's too much checks

Reply:
I think the InvalidVersionWrapper is ok, just I think the function should first check the simplest case where the wrapper name is correct then move to the next case rather than we check the simplest case currently last